### PR TITLE
feat(contracts): replaces active_contracts.json with @aragon/osx-comons-configs

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

feat(contracts): replaces active_contracts.json with @aragon/osx-comons-configs
fix(contracts): align naming of contract during deployment with configs package
fix(contracts): renames managingDAO for deployment to managementDAO

Depends on pull https://github.com/aragon/osx-commons/pull/44

Task ID: [OS-930](https://aragonassociation.atlassian.net/browse/OS-930)

## Type of change

See the framework lifecylce in `packages/contracts/docs/framework-lifecycle` to decide what kind of change this pull request is.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [ ] I have updated the `UPDATE_CHECKLIST` file in the root folder.

[OS-930]: https://aragonassociation.atlassian.net/browse/OS-930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Rekard0 — APPROVED** [review]

**Review by Rekard0 — DISMISSED** [review]
LGTM

**Review by Rekard0 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ In a nutshell, each DAO is composed of 3 interconnecting components:
 
 1. **The DAO contract:** The DAO contract is where the **core functionality** of the DAO lies. It is the contract in charge of:
    - Representing the identity and metadata of the DAO (ENS name, logo, description, other metadata)
-   - Holding and managing the treasury assets
+   - Holding and management the treasury assets
```
Comment (Michael-A-Heuer) [inline README.md:111]: wrong replacement

File: README.md [hunk 2]
```diff
@@ -192,34 +192,7 @@ You can find more details about [our deployment checklist here](https://github.c
...
-**Mainnets**
-
-1. Deploy the new contracts version to both mainnets and testnets
-2. Update the `active_contracts.json` file on `main`
-3. Create a **draft** release (e.g. `1.3.0`) on GitHub
-4. Wait for the managing DAO to approve on mainnets and testnets
-5. Update the NPM package version (see [Releasing NPM packages](#releasing-npm-packages))
-   - Create a PR to `main` with the release labels
-6. Unmark the Release as a draft and deploy the NPM packages
+Follow [our update cheklist here](https://github.com/aragon/osx/blob/develop/UPDATE_CHECKLIST.md).
```
Comment (Michael-A-Heuer) [inline README.md:195]:
```suggestion
Follow [our update checklist here](https://github.com/aragon/osx/blob/develop/UPDATE_CHECKLIST.md).
```

File: UPDATE_CHECKLIST.md [hunk 3]
```diff
@@ -42,13 +42,13 @@ To update run `yarn deploy --network NETWORK` in `packages/contracts` and replac
 ### Configuration updates
 
 - [ ] Take the addresses from this file `packages/contracts/deployed_contracts.json`
-- [ ] Update `active_contracts.json` with the new deployed addresses
+- [ ] Update the deployment in the `@aragon/osx-commons-config` package found here: [https://github.com/aragon/osx-commons/tree/develop/configs](https://github.com/aragon/osx-commons/tree/develop/configs)
 - [ ] Update `packages/contracts/Releases.md` with the new deployed addresses
 - [ ] Add a Github Release with the version number as tag and the defined content (check previous releases for reference)
 
-### ManagingDAO
+### ManagementDAO
```
Comment (Michael-A-Heuer) [inline UPDATE_CHECKLIST.md:49]:
```suggestion
### Management DAO
```

File: README.md (hunk 1 above)
Comment (mathewmeconry) [reply README.md:111]: done

File: README.md (hunk 2 above)
Comment (mathewmeconry) [reply README.md:195]: done

File: README.md (hunk 1 above)
Comment (mathewmeconry) [reply README.md:111]: done

File: UPDATE_CHECKLIST.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mathewmeconry) [reply UPDATE_CHECKLIST.md:49]: better?

File: packages/contracts/deploy/new/10_framework/02_ens_subdomain_registrars.ts [hunk 4]
```diff
@@ -63,15 +66,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
...
           methodName: 'initialize',
-          args: [managingDAOAddress, ensRegistryAddress, pluginNode],
+          args: [managementDAOAddress, ensRegistryAddress, pluginNode],
         },
       },
     },
   });
 
-  // Get PluginRepoRegistry's `ENSSubdomainRegistrar` contract.
+  // Get PluginRepoRegistry's `PluginENSSubdomainRegistrarProxy` contract.
```
[the remark this answers is not in the document]
Comment (mathewmeconry) [reply packages/contracts/deploy/new/10_framework/02_ens_subdomain_registrars.ts:75]: done

**Review by Michael-A-Heuer — APPROVED** [review]

**Review by Rekard0 — APPROVED** [review]

**Pull request merged**